src/definition/fetch_test.go

```diff
@@ -44,15 +43,15 @@ func TestFromRawErrorsOnNotFound(t *testing.T) {
 	}
 
 	v, err := FromRaw("metric_name_3")("nonExistingGroup", "entity2", raw)
-	assert.EqualError(t, err, "group not found")
+	assert.EqualError(t, err, "group \"nonExistingGroup\" not found")
 	assert.Nil(t, v)
 
 	v, err = FromRaw("metric_name_3")("group1", "nonExistingEntity", raw)
-	assert.EqualError(t, err, "entity not found")
+	assert.EqualError(t, err, "entity \"nonExistingEntity\" not found")
 	assert.Nil(t, v)
 
 	v, err = FromRaw("non_existing_metric")("group1", "entity2", raw)
-	assert.EqualError(t, err, "metric not found")
+	assert.EqualError(t, err, "metric \"non_existing_metric\" not found")
 	assert.Nil(t, v)
```

Comment:
Could we table-test this? Seems like a quick change.

Reply:
I'll mark this comment as duplicate to the other one.